services/workflows-service/src/alert/dtos/assign-alert.dto.ts

```diff
@@ -0,0 +1,81 @@
+import { Type } from 'class-transformer';
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  ArrayMinSize,
+  IsArray,
+  IsString,
+  MinLength,
+  ValidateNested,
+  minLength,
+} from 'class-validator';
+import { BulkStatus } from '../types';
+import type { TAlertUpdateResponse, TBulkStatus } from '../types';
+
+export class AlertsIdsByProjectDto {
+  @ApiProperty({
+    type: Array<String>,
+  })
+  @Type(() => Array<String>)
+  @IsArray()
+  @IsString({ each: true })
+  @ArrayMinSize(1)
+  alertIds!: string[];
+
+  @ApiProperty({
+    type: String,
+    required: true,
+  })
+  @Type(() => String)
+  @IsString()
+  @MinLength(1)
+  projectId!: string;
+}
+
+export class AlertAssigneeUniqueDto {
+  @ApiProperty({
+    required: true,
+    type: String,
+  })
+  @IsString()
+  assigneeId!: string;
+}
+
+export class AlertUpdateResponse {
+  @ApiProperty({
+    required: true,
+    type: String,
+  })
+  alertId!: string;
+
+  @ApiProperty({
+    required: true,
+    type: String,
+  })
+  status!: string;
+}
+
+export class BulkAssignAlertsResponse {
+  @ApiProperty({
+    required: true,
+    enum: BulkStatus,
+  })
+  overallStatus!: TBulkStatus;
```

Comment:
@alonp99 did we agree that the HTTP code can be used over `overallStatus`?

Reply:
this schema provided by @alonp99 
let me know what status code would you expect to ?

Reply:
We can use only status code or both status code and overallStatus

Reply:
Can you explain what status code would you expect when overallStatus is `partial`?